apps/ag-grid-storybook/src/app/ag-grid/ag-grid-stories.component.scss

```diff
@@ -0,0 +1,43 @@
+:root {
```

Comment:
looks like there is more styling here than the typical storybook. Did you run into any gotchas that would be worth noting in the e2e testing docs? Seems like its pretty ag grid specific so maybe not

Reply:
When not using auto-height, AG Grid needs a height style. With or without auto-height, setting exact sizes helped minimize the overflow displayed in Percy. Otherwise Percy attempts to find the right viewport size.